src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,81 @@
-export const Pagination = () => {};
+import React, { Dispatch, SetStateAction } from 'react';
+
+type Props = {
+  currentPage: number;
+  total: number;
+  itemsPerPage: number;
+  onPageChange: (page: number) => void;
+  setCurrentPage: Dispatch<SetStateAction<number>>;
+};
+
+export const Pagination: React.FC<Props> = ({
+  currentPage,
+  total,
+  itemsPerPage,
+  onPageChange,
+  setCurrentPage,
```

Comment:
The `setCurrentPage` prop is being destructured but is not needed. The `onPageChange` function should be used for changing pages instead.